qctrlopencontrols/dynamic_decoupling_sequences/driven_controls.py

```diff
@@ -206,6 +206,16 @@ def convert_dds_to_driven_control(
         azimuthal_angles = np.append(azimuthal_angles, [0])
         detuning_rotations = np.append(detuning_rotations, [0])
 
+    # check that the offsets are correctly sorted in time
+    time_differences = np.diff(offsets)
+    if not np.all(np.logical_or(np.greater(time_differences, 0.),
```

Comment:
In this case I'm not convinced that we should allow "close" offsets. Typically we'd use a "close" check when equality is allowed, but here I think we want the offsets to be strictly increasing rather than non-decreasing.